sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java

```diff
@@ -93,42 +94,42 @@ public SettingSelector setKeys(String... keys) {
      * <p>
      * Examples:
      * <ul>
-     * <li>If {@code labels = "*"}, settings with any label are returned.</li>
-     * <li>If {@code labels = "\0"}, settings without any label are returned.</li>
-     * <li>If {@code labels = ""}, settings without any label are returned.</li>
-     * <li>If {@code labels = null}, settings without any label are returned.</li>
-     * <li>If {@code labels = "abc1234"}, settings with a label equal to "abc1234" are returned.</li>
-     * <li>If {@code labels = "abc*"}, settings with a label starting with "abc" are returned.</li>
-     * <li>If {@code labels = "*abc*"}, settings with a label containing "abc" are returned.</li>
-     * <li>If {@code labels = "abc,def"}, settings with labels "abc" or "def" are returned.</li>
+     * <li>If {@code labelFilter = "*"}, settings with any label are returned.</li>
+     * <li>If {@code labelFilter = "\0"}, settings without any label are returned.</li>
+     * <li>If {@code labelFilter = ""}, settings without any label are returned.</li>
+     * <li>If {@code labelFilter = null}, settings without any label are returned.</li>
+     * <li>If {@code labelFilter = "abc1234"}, settings with a label equal to "abc1234" are returned.</li>
+     * <li>If {@code labelFilter = "abc*"}, settings with a label starting with "abc" are returned.</li>
+     * <li>If {@code labelFilter = "*abc*"}, settings with a label containing "abc" are returned.</li>
+     * <li>If {@code labelFilter = "abc,def"}, settings with labels "abc" or "def" are returned.</li>
      * </ul>
      *
      * @return labels The labels used to filter GET requests from the service.
      */
-    public String[] getLabels() {
-        return labels == null ? new String[0] : CoreUtils.clone(labels);
+    public String getLabelFilter() {
+        return labelFilter == null ? "" : labelFilter;
```

Comment:
An empty string seems different than a collection with no items in it. Is this correct? It may account for why those JSON files are breaking.

Reply:
Thanks. It should be the reason

Reply:
Is this the correct behaviour, is what I am wondering